[BUG] 스케줄링 서버 쓰레드풀 고갈

### 관련 로그
```
2025-01-06 10:18:00 ERROR [scheduling-1] o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.springframework.core.task.TaskRejectedException: ExecutorService in active state did not accept task: org.springframework.aop.interceptor.AsyncExecutionInterceptor$$Lambda$2174/0x00000008019acdd0@374d7f06
        at org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor.submit(ThreadPoolTaskExecutor.java:387)
        at org.springframework.aop.interceptor.AsyncExecutionAspectSupport.doSubmit(AsyncExecutionAspectSupport.java:297)
        at org.springframework.aop.interceptor.AsyncExecutionInterceptor.invoke(AsyncExecutionInterceptor.java:127)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184)
        at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:765)
        at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:717)
        at com.finance.adam.scheduler.ScheduledTasks$$SpringCGLIB$$0.recentReportRedisUpdate(<generated>)
        at jdk.internal.reflect.GeneratedMethodAccessor104.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
        at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
        at io.micrometer.observation.Observation.observe(Observation.java:499)
        at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
        at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
        at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1135)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:831)
Caused by: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@603a3cac[Not completed, task = org.springframework.aop.interceptor.AsyncExecutionInterceptor$$Lambda$2174/0x00000008019acdd0@374d7f06] rejected from org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor$1@47c2849b[Running, pool size = 50, active threads = 50, queued tasks = 500, completed tasks = 1017]
        at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2064)
        at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:832)
        at java.base/java.util.concurrent.ThreadPoolExecutor.execute(ThreadPoolExecutor.java:1364)
        at org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor$1.execute(ThreadPoolTaskExecutor.java:269)
        at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:145)
        at org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor.submit(ThreadPoolTaskExecutor.java:384)
        ... 21 common frames omitted
```

--- a/adam/src/main/java/com/finance/adam/service/ScreeningService.java
+++ b/adam/src/main/java/com/finance/adam/service/ScreeningService.java
@@ -39,7 +39,7 @@ public class ScreeningService {
 
             StockPrice stockPrice = corpInfo.getStockPrice();
             if(stockPrice == null){
-                log.error("StockPrice is null for corporation: corpCode={}, stockCode={}", corpInfo.getCorpCode(), corpInfo.getStockCode());
+                log.info("StockPrice is null for corporation: corpCode={}, stockCode={}", corpInfo.getCorpCode(), corpInfo.getStockCode());
                 continue;
             }
             stockInfoDTO.setOpeningPrice(stockPrice.getOpeningPrice());

